tests/signal_handlers/SIGSEGV/fuzz.js

```diff
@@ -36,3 +38,22 @@ module.exports.SIGSEGV_ASYNC = (data) => {
 	}
 	i++;
 };
+
+module.exports.NATIVE_SIGSEGV_SYNC = (data) => {
+	if (i === 1000) {
```

Comment:
As the "execute on Xs invocation" is now used in multiple tests a dedicated function to abstract that away could be useful.

Reply:
Fair enough :+1: